src/cpp/DataBroker.cpp

```diff
@@ -329,10 +329,17 @@ bool DataBroker::run_time(
     return true;
 }
 
-// TODO decide default qos
 eprosima::fastdds::dds::DomainParticipantQos DataBroker::default_participant_qos()
```

Comment:
This function as `wan_participant_qos` could be moved to either `DataBrokerParticipant` and  `DataBrokerWANParticipant`

Reply:
Addressed in 1bdaa59